libimage/runtime.go

```diff
@@ -408,17 +409,15 @@ func (r *Runtime) lookupImageInDigestsAndRepoTags(name string, options *LookupIm
 				}
 			}
 		}
-
-		return nil, "", errors.Wrap(storage.ErrImageUnknown, name)
 	}
 
 	// Podman compat: if we're looking for a short name but couldn't
-	// resolve it via the registries.conf dance, we need to look at *all*
-	// images and check if the name we're looking for matches a repo tag.
-	// Split the name into a repo/tag pair
+	// resolve until this point, we need to look at *all* images and check
+	// if the name we're looking for matches a repo tag.  Split the name
+	// into a repo/tag pair
 	split := strings.SplitN(name, ":", 2)
 	repo := split[0]
-	tag := ""
+	tag := "latest" // See containers/podman/issues/11964.
 	if len(split) == 2 {
 		tag = split[1]
 	}
```

Comment:
This is the only caller of `inRepoTags`; now that `tag` can not be empty, it can be simplified.